scripts/sidebar/index.js

```diff
@@ -3,16 +3,29 @@ import "./style.scss";
 import { Fragment } from "@wordpress/element";
 import { registerPlugin } from "@wordpress/plugins";
 
-import Sidebar from "./components/sidebar";
-import MenuItem from "./components/menu-item";
+import SidebarUnsplash from "./components/sidebar/unsplash";
+import SidebarGiphy from "./components/sidebar/giphy";
+import MenuItemUnsplash from "./components/menu-item/unsplash";
+import MenuItemGiphy from "./components/menu-item/giphy";
 
-const DropitPlugin = () => (
+const Dropit = () => (
   <Fragment>
-    <Sidebar />
-    <MenuItem />
+    <SidebarUnsplash />
+    <MenuItemUnsplash />
   </Fragment>
 );
 
 registerPlugin("dropit", {
```

Comment:
I believe instead of registering two plugins, we could use the same `registerPlugin` call for both

Reply:
yes, as I said really basic merge, need enhancements